### ISSUE TITLE
Mobile keyboard caused orientation bug

### ISSUE THREAD

**Issue by bit-captain** [duskload/react-device-detect#97]
So the orientation worked fine on most of the mobile phone but there is some mobile that caused viewport to change when user tried to type something and their keyboard popup for example input box then it created a bug . Could you please fix this issue pls?

**Comment by duskload** [comment]
@bit-captain What exactly is the bug here? Showing keyboard on mobile device when you interacting with input fields is a common behavior and can be handled from your code (showing hiding the keyboard)

If you still believe it's a bug please provide some reproducible example of it.

**Comment by duskload** [comment]
Closing the issue, please feel free to reopen this or file another issue, but provide reproducible example of it.

**Comment by iggyZiggy** [comment]
Steps to reproduce this issue:
1) use input field
2) use isPortrait and withOrientationChange like in example
3) once keyboard shows up, isPortrait is set to false

**Comment by vomc** [comment]
I have also discovered this recently, will look to make some kind of reproducible example. In my case this was hard to discover as this only happens on Android, where showing the soft keyboard causes a window.resize event to fire which sets `isPortrait` to false. This does not happen with the soft keyboard on ios as that does not trigger a window.resize event.

**Comment by nichita-pasecinic** [comment]
@duskload Hi, 
you can find the steps to reproduce [in this comment](https://github.com/duskload/react-device-detect/issues/97#issuecomment-721119349)

TL;DR: Just open a input on mobile in portrait mode and `isLandscape` will became `true`, though it should not
